build/__tools__/clean.sh

```diff
@@ -29,8 +29,8 @@ yell --------------- COMMON ---------------
 cd $SRCDIR/common/crypto/verify_ias_report
 rm -f ias-certificates.cpp
 
-cd $SRCDIR/common/interpreter/gipsy_scheme/packages
-rm -f package.h package.scm
+#cd $SRCDIR/common/interpreter/gipsy_scheme/packages
```

Comment:
they are no longer auto generated. will be part of the repo. Party of the fix for tiny scheme issue that we were facing.

Reply:
they should but they can't be regenerated because of a garbage collection bug in the stock tinyscheme. this should just be a temporary fix until we can completely remove tinyscheme.